packaging/hudi-integ-test-bundle/pom.xml

```diff
@@ -319,12 +319,19 @@
     <dependency>
       <groupId>com.fasterxml.jackson.module</groupId>
       <artifactId>jackson-module-scala_${scala.binary.version}</artifactId>
+      <version>${fasterxml.jackson.module.scala.version}</version>
     </dependency>
 
     <dependency>
       <groupId>com.fasterxml.jackson.dataformat</groupId>
       <artifactId>jackson-dataformat-yaml</artifactId>
-      <version>2.7.4</version>
+      <version>${fasterxml.spark3.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.fasterxml.jackson.datatype</groupId>
+      <artifactId>jackson-datatype-jsr310</artifactId>
+      <version>${fasterxml.spark3.version}</version>
```

Comment:
Do we need to include the `version` tag?

Reply:
It seems better to explicitly specify a `<version>`, so that was its clear what version is being included and one doesn't have to manually dig down to try to discover which version was included if any issue comes up like the one we are dealing with here.

Reply:
in the CI setup for integ test, integ test bundle was only running for spark 2, that's why the jackson version is set to 2.7.4 here

Reply:
Based on offline discussion, I modified this to `<version>${fasterxml.version}</version>` to pick up the right jackson package version for a given spark version.

Reply:
Also using `${fasterxml.jackson.module.scala.version}` and `${fasterxml.jackson.dataformat.yaml.version}` to pull in the appropriate version for `jackson-module-scala_${scala.binary.version}` and `jackson-dataformat-yaml`.